upcase_booklist_app/data.py

```diff
@@ -0,0 +1,38 @@
+
+books = {
```

Comment:
Great thought on moving this to another module. If I were you, I'd create a new directory and rename this to `upcase_booklist_app/data/books.py`, that way all your data, like authors, will live in the same directory.